modules/web-console/frontend/app/modules/agent/AgentManager.service.js

```diff
@@ -692,7 +692,7 @@ export default class AgentManager {
      * @param {Number} pageSize
      * @param {Boolean} [lazy] query flag.
      * @param {Boolean} [collocated] Collocated query.
-     * @returns {Promise<VisorQueryResult>}
+     * @returns {Promise.<{{responseNodeId: String, queryId: String, columns: String[], rows: {List.<Object[]>}, hasMore: Boolean, duration: Number}}>} Query execution result.
```

Comment:
1. You can reuse type definitions using @typedef or import from TS file. Ask me how to.
2. Surely there's no `List` in parsed JSON. Please use correct types.

Reply:
Fixed jsdock.